.nvmrc

```diff
@@ -0,0 +1 @@
+v19.7.0
```

Comment:
I think we should use the [LTS](https://nodejs.org/en/download) (18) instead of this one